frontend/src/editor/transcription_editor.tsx

```diff
@@ -49,6 +52,10 @@ function renderElement({ element, children, attributes }: RenderElementProps): J
             >
               {formattedTime(startAtom.start)}
             </div>
+            <div
+              className="absolute right-0 top-0 w-2 h-full rounded-md mr-2 -mt-0.5"
+              style={element.speaker ? { backgroundColor: speakerColors[element.speaker] } : {}}
```

Comment:
Do we actually want to use `backgroundColor`? How does this interact with the text color? 
What about some kind of colored bar next to the paragraph?